### PR TITLE
Data Output

### PR DESCRIPTION
**Pull request by viridispirus**
#159 fixed.

### CODE REVIEW

File: doc/Requirements Specification/Parts/Data.lyx [hunk 1]
```diff
@@ -424,7 +424,20 @@ glsuserii{internal action}
...
- execution time.
+ execution time and the CPU 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+glspl{resource demand}
```
Comment (acBerger) [inline doc/Requirements Specification/Parts/Data.lyx]: in the PDF is written "resouce demandss" with double "s". I think it is defined as "resource demands" so use gls (or change definition to "resource demand".

**Comment by acBerger** [comment]
Do we want to have both, execution time and resource demands in output or only resource demands?

**Comment by acBerger** [comment]
[PDF] (http://viridispirus.github.io/Beagle/branches/DataOutput/Requirements%20Specification.pdf)

**Comment by viridispirus** [comment]
Why not? Is it circuitous if we have both?

**Comment by flyfl** [comment]
You have made changes at the wrong location, I guess. It was ment to specify "internal actions" as CPU resource demands. The new sentence should be like "The execution time of the software components' CPU recource demands"

**Comment by jGleitz** [comment]
I guess you can’t say resource demands have an execution time. I’d prefer to simply state “The software’s components’ CPU resource demands”

**Comment by viridispirus** [comment]
@jGleitz Ok I see. And do you think we don't need the execution time of the components?

**Comment by jGleitz** [comment]
> And do you think we don't need the execution time of the components?

There is no such thing as a component’s execution time. Components have internal actions that have resource demands. Only when knowing the component’s usage, deployment and assembly context, you can predict (or measure) the execution time of one of its services. See [Krogmann 2011] for details, Chapter 5 has a detailed introduction on this.

**Comment by viridispirus** [comment]
Ok, thanks for explanation.

**Comment by acBerger** [comment]
lgtm

**Comment by jGleitz** [comment]
lgtm

**Pull request merged**